test/suites/drift/suite_test.go

```diff
@@ -224,6 +224,46 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			g.Expect(node.Annotations).To(HaveKeyWithValue(v1alpha5.VoluntaryDisruptionAnnotationKey, v1alpha5.VoluntaryDisruptionDriftedAnnotationValue))
 		}).Should(Succeed())
 
+		delete(pod.Annotations, v1alpha5.DoNotEvictPodAnnotationKey)
+		env.ExpectUpdated(pod)
+		env.EventuallyExpectNotFound(pod, node)
+	})
+	It("should deprovision nodes that have drifted due to subnets", func() {
+		env.ExpectSettingsOverridden(map[string]string{
```

Comment:
Should this be in the `BeforeEach`?

Reply:
Ah it already is, must've been from a real old rebase. good catch!